src/Microsoft.Diagnostics.EventFlow.FilterParserGenerator/Microsoft.Diagnostics.EventFlow.FilterParserGenerator.csproj

```diff
@@ -32,6 +32,10 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.Diagnostics.Correlation.Common, Version=1.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
```

Comment:
Is it really necessary for the FilterParserGenerator to take a dependency on Correlation lib?